README.md

```diff
@@ -220,6 +220,16 @@ nelmio_security:
         compat_headers: false
 ```
 
+(Optional) Use *enable_nonce* to add a nonce value to your every request.
+
+```yaml
+nelmio_security:
+    csp:
+        enable_nonce: true
+```
+
+The nonce token that was used in the current request can be retrieved by using the *nelmio_security.nonce_generator* service.
```

Comment:
It'd be nice to have an example of how this looks especially with the ->getCurrentToken call so people don't have to dig in the code to see which method should be called.